### ISSUE TITLE
Color Contrast Ratio for the text 'Month-year' in graph is 1.917:1 which is less than expected (4.5:1)

### ISSUE THREAD

**Issue by msftedad** [highcharts/highcharts#21512]
#### Expected behaviour
Contrast ratio of 'Month & year' present for 'High chart' graph should be greater than or equal to '4.5:1' with the background.

#### Actual behaviour
Contrast ratio of 'Month & year' present for 'High chart' graph is '3.4:1' with the background, which is less than '4.5:1'(required).

#### Live demo with steps to reproduce

1.Open the URL [Try Out Highcharts Interactive Stock GUI | Highcharts](https://www.highcharts.com/demo/stock/stock-tools-gui) in edge dev browser.
2.'Try out Highcharts Interactive Stock GUI' page willopen.
3.Navigate to the graph which contains trim handles just above the 'Code pen' button.
4.Observe whether the contrast ratio of 'Month & year' present for 'High chart' graph using accessibility insights for windows.

<img width="931" alt="Contrast ratio fails for Month  amp; year of high chart" src="https://github.com/highcharts/highcharts/assets/93735775/9858cf2b-5a98-44e6-bb57-cc180d7ccf36">

<img width="919" alt="Note 1-Contrast ratio fails for text present below the graph" src="https://github.com/highcharts/highcharts/assets/93735775/3f51bde3-df33-44f5-a8d3-586e1f4e556f">

#### Product version

Highcharts: 11

#### Affected browser(s)
OS: Win11 version 24H2 (OS build 26244.5000)
Brower: Edge Dev version 128.0.2677.1 (Official build) dev (64-bit)

**Comment by karolkolodziej** [comment]
Hi @msftedad!

Thank you for pointing that out!

**Comment by msftedad** [comment]
Thanks for the update. We will share this workaround with Product team.

**Comment by msftedad** [comment]
We have closed this issue at our end. Please close this. Thanks
